CUDADataFormats/SiPixelCluster/interface/gpuClusteringConstants.h

```diff
@@ -13,9 +13,9 @@ namespace gpuClustering {
   // tested on MC events with 55-75 pileup events
   constexpr uint32_t maxHitsInIter() { return 160; }  //TODO better tuning for PU 140-200
 #endif
-  constexpr uint32_t maxHitsInModule() { return 1024; }
+  constexpr uint32_t maxHitsInModule() { return 2048; }
```

Comment:
Does this affect all workflows (Phase-1, Phase-2, HIons) ?
Is it intended ?

Reply:
This is basically not used anymore and replaced by `TrackeTraits::maxNumClustersPerModules` everywhere. Could be removed.

Reply:
OK... then it would be better to remove it :)